### ISSUE TITLE
Whole cache warm job fails if 1 page fails to load

### ISSUE THREAD

**Issue by riasvdv** [putyourlightson/craft-blitz#42]
The whole cache warm job fails when 1 page fails to load (when it gives a 500, 503, ... for whatever reason)

Is it possible to catch those errors, log them somewhere, but continue with the job to other pages?

Error in queue table:
```
Server error: `GET https://www.mysite.com/blog/some-blogpost` resulted in a `500 Internal Server Error` response:
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="nl-BE">
<head>
        <meta http-equiv="X-UA-Compatibl (truncated...)

```

**Comment by putyourlightson** [comment]
This is fixed in the upcoming version, available in the develop branch in case you need it asap.
https://github.com/putyourlightson/craft-blitz/tree/develop

**Comment by riasvdv** [comment]
Ooh the concurrency looks good, hadn't seen that 👍

**Comment by putyourlightson** [comment]
Yup, it makes warming the cache crazy fast!!

**Comment by putyourlightson** [comment]
Fixed in 1.8.0.
